extensions-core/core/runtime/src/main/java/org/apache/camel/quarkus/core/CamelConfig.java

```diff
@@ -62,6 +62,13 @@
         @ConfigItem(defaultValue = "true")
         public boolean enabled;
 
+        /**
+         * If {@code true}, the camel context will be started at initialization time
+         * to boot up faster.
+         */
+        @ConfigItem
+        public boolean lightweight;
```

Comment:
I think the flag name does not express well what it does. How about `staticInitStart` or `startInStaticInit`?

Reply:
I'll add a flag for that later.

Reply:
> I think the flag name does not express well what it does. How about `staticInitStart` or `startInStaticInit`?

Taking back, now that I understand better what the PR does. The option name is fine. Sorry for the confusion.